--- a/src/FlightMap/Widgets/QGCArtificialHorizon.qml
+++ b/src/FlightMap/Widgets/QGCArtificialHorizon.qml
@@ -20,8 +20,8 @@
             smooth: true
             antialiasing: true
             gradient: Gradient {
-                GradientStop { position: 0.25; color: Qt.hsla(0.6, 1.0, 0.25) }
-                GradientStop { position: 0.5;  color: Qt.hsla(0.6, 0.5, 0.55) }
+                GradientStop { position: 0.25; color: "#CC151B24" }
+                GradientStop { position: 0.5;  color: "#CC222A36" }
             }
         }
         Rectangle {
@@ -35,8 +35,8 @@
             smooth: true
             antialiasing: true
             gradient: Gradient {
-                GradientStop { position: 0.0;  color: Qt.hsla(0.25,  0.5, 0.45) }
-                GradientStop { position: 0.25; color: Qt.hsla(0.25, 0.75, 0.25) }
+                GradientStop { position: 0.0;  color: "#CC1A241B" }
+                GradientStop { position: 0.25; color: "#CC0E140F" }
             }
         }
         transform: [
--- a/src/FlightMap/Widgets/QGCAttitudeWidget.qml
+++ b/src/FlightMap/Widgets/QGCAttitudeWidget.qml
@@ -108,8 +108,8 @@
         anchors.fill:   parent
         radius:         width / 2
         color:          Qt.rgba(0,0,0,0)
-        border.color:   qgcPal.text
-        border.width:   1
+        border.color:   "#40FFFFFF"
+        border.width:   1.5
     }
 
     QGCLabel {
--- a/src/Toolbar/MainStatusIndicator.qml
+++ b/src/Toolbar/MainStatusIndicator.qml
@@ -102,7 +102,7 @@
                     }
                 }
             } else {
-                _mainStatusBGColor = qgcPal.brandingPurple
+                _mainStatusBGColor = "#0B0E14"
                 return mainStatusLabel._disconnectedText
             }
         }
