--- a/src/QmlControls/SimpleItemEditor.qml
+++ b/src/QmlControls/SimpleItemEditor.qml
@@ -59,7 +59,7 @@
         QGCLabel {
             width:          parent.width
             wrapMode:       Text.WordWrap
-            font.pointSize: ScreenTools.smallFontPointSize
+            font.pointSize: ScreenTools.defaultFontPointSize
             text:           missionItem.rawEdit ?
                                 qsTr("Provides advanced access to all commands/parameters. Be very careful!") :
                                 missionItem.commandDescription
@@ -124,7 +124,7 @@
                 QGCLabel {
                     Layout.fillWidth:   true
                     wrapMode:           Text.WordWrap
-                    font.pointSize:     ScreenTools.smallFontPointSize
+                    font.pointSize:     ScreenTools.defaultFontPointSize
                     text:               qsTr("Altitude below specifies the approximate altitude of the ground. Normally 0 for landing back at original launch location.")
                     visible:            missionItem.isLandCommand
                 }
@@ -154,7 +154,7 @@
                         QGCLabel {
                             Layout.alignment:   Qt.AlignBaseline
                             text:               qsTr("Altitude")
-                            font.pointSize:     ScreenTools.smallFontPointSize
+                            font.pointSize:     ScreenTools.defaultFontPointSize
                         }
                         QGCLabel {
                             id:                 altModeLabel
@@ -178,7 +178,7 @@
                 }
 
                 QGCLabel {
-                    font.pointSize:     ScreenTools.smallFontPointSize
+                    font.pointSize:     ScreenTools.defaultFontPointSize
                     text:               qsTr("Actual AMSL alt sent: %1 %2").arg(missionItem.amslAltAboveTerrain.valueString).arg(missionItem.amslAltAboveTerrain.units)
                     visible:            missionItem.altitudeMode === QGroundControl.AltitudeModeCalcAboveTerrain
                 }
@@ -197,7 +197,7 @@
                         spacing:            0
 
                         QGCLabel {
-                            font.pointSize: ScreenTools.smallFontPointSize
+                            font.pointSize: ScreenTools.defaultFontPointSize
                             text:           object.name
                             visible:        object.name !== ""
                         }
--- a/src/QmlControls/MissionSettingsEditor.qml
+++ b/src/QmlControls/MissionSettingsEditor.qml
@@ -66,7 +66,7 @@
 
         QGCLabel {
             text:           qsTr("All Altitudes")
-            font.pointSize: ScreenTools.smallFontPointSize
+            font.pointSize: ScreenTools.defaultFontPointSize
         }
         MouseArea {
             Layout.preferredWidth:  childrenRect.width
@@ -113,7 +113,7 @@
 
         QGCLabel {
             text:           qsTr("Initial Waypoint Alt")
-            font.pointSize: ScreenTools.smallFontPointSize
+            font.pointSize: ScreenTools.defaultFontPointSize
         }
         FactTextField {
             fact:               QGroundControl.settingsManager.appSettings.defaultMissionItemAltitude
@@ -158,7 +158,7 @@
                 text:                   qsTr("Above camera commands will take affect immediately upon mission start.")
                 wrapMode:               Text.WordWrap
                 horizontalAlignment:    Text.AlignHCenter
-                font.pointSize:         ScreenTools.smallFontPointSize
+                font.pointSize:         ScreenTools.defaultFontPointSize
                 visible:                _showCameraSection && cameraSection.checked
             }
 
@@ -216,7 +216,7 @@
                     Layout.alignment:       Qt.AlignHCenter
                     Layout.fillWidth:       true
                     wrapMode:               Text.WordWrap
-                    font.pointSize:         ScreenTools.smallFontPointSize
+                    font.pointSize:         ScreenTools.defaultFontPointSize
                     text:                   qsTr("The following speed values are used to calculate total mission time. They do not affect the flight speed for the mission.")
                     visible:                _showCruiseSpeed || _showHoverSpeed
                 }
@@ -278,7 +278,7 @@
                 QGCLabel {
                     width:                  parent.width
                     wrapMode:               Text.WordWrap
-                    font.pointSize:         ScreenTools.smallFontPointSize
+                    font.pointSize:         ScreenTools.defaultFontPointSize
                     text:                   qsTr("Actual position set by vehicle at flight time.")
                     horizontalAlignment:    Text.AlignHCenter
                 }
