--- a/translations/qgc_json_no_NO.ts
+++ b/translations/qgc_json_no_NO.ts
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="no" sourcelanguage="en">
+<TS version="2.1" language="nb_NO" sourcelanguage="en">
 <context>
     <name>APMAirframeComponent</name>
     <message>
--- a/translations/qgc_source_no_NO.ts
+++ b/translations/qgc_source_no_NO.ts
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="no" sourcelanguage="en">
+<TS version="2.1" language="nb_NO" sourcelanguage="en">
 <context>
     <name>APMAirframeComponent</name>
     <message>
